### PR TITLE
Make a troubelshooting page for tutorial authoring

### PR DESCRIPTION
**Pull request by ganicke**
Here's is a start on a troubleshooting page for tutorial writing. Suggest other tips to include.

Closes https://github.com/microsoft/pxt-arcade/issues/6408

### CODE REVIEW

**Review by Jaqster — APPROVED** [review]
Looks like a great start!  I'm going to try and crowdsource contributions for this...

File: docs/writing-docs/tutorials/troubleshooting.md
Comment (kiki-lee) [inline docs/writing-docs/tutorials/troubleshooting.md:1]: Some other thoughts:
- Dashes '-' in your image pack repo name need to disappear when you import the pack into a tutorial.  For instance "my-images" becomes "my images" but "my_images" stays the same. 
- If you added a "kind" the way that our compiler does, it won't work correctly in the tutorial. You need to add //% isKind  before each declaration.
- If the game screen on the tutorial seems to spin and never load, there's probably an error. Start by opening the JS console and trying to re-run the tutorial. It should give you an idea of what's making it hang.

Comment (ganicke) [reply docs/writing-docs/tutorials/troubleshooting.md:1]: Thanks! I added those in.

**Review by kiki-lee — CHANGES_REQUESTED** [review]
No spaces. That would turn it into three separate variables in the code.  It will just be `myassetpack`.

**Comment by ganicke** [comment]
> No spaces. That would turn it into three separate variables in the code. It will just be `myassetpack`.

OK. I was confused cuz you mentioned earlier:

Dashes '-' in your image pack repo name need to disappear when you import the pack into a tutorial. For instance "my-images" becomes "my images" but "my_images" stays the same.

**Review by kiki-lee — APPROVED** [review]

**Pull request merged**